lemarche/tenders/models.py

```diff
@@ -439,7 +439,7 @@ def location_display(self) -> str:
             return self.perimeters_list_string
 
     def questions_list(self):
-        return self.questions.values_list("text", flat=True)
+        return self.questions.values("id", "text")
```

Comment:
```suggestion
        return list(self.questions.values("id", "text"))
```
ca va renvoyer un queryset sinon